lib/src/components/NxDropdown/NxDropdown.scss

```diff
@@ -8,57 +8,80 @@
 @import '../../scss-shared/nx-text-helpers';
 @import '../../scss-shared/nx-container-helpers';
 
+$nx-dropdown-width: 250px;
+
 .nx-dropdown {
   display: inline-block;
   position: relative;
+  width: $nx-dropdown-width;
 }
 
 .nx-dropdown__toggle {
+  display: flex;
   margin: 0;
   outline: none;
+  text-align: left;
+  padding-right: $nx-spacing-md;
+  padding-left: $nx-spacing-md;
+  width: $nx-dropdown-width;
+
+  span {
+    flex-grow: 1;
+  }
+
+  .nx-icon {
+    align-self: center;
+    width: 12px;
+  }
 }
 
 .nx-dropdown-menu {
   @include container-vertical();
 
   background: $nx-off-white;
   border: $nx-border-dark;
-  border-radius: 4px;
-  box-shadow: -4px 4px 10px 0 $nx-slate;
+  border-radius: $nx-border-radius;
+  box-shadow: 0 2px 12px 0 rgba(74, 77, 113, 0.2),
+              0 1px 2px 0 rgba(91, 95, 141, 0.2),
+              0 1px 1px 0 rgba(124, 128, 179, 0.2);
   color: $nx-text-color-light;
   display: block;
-  margin: 6px 0 0 0;
-  max-width: 300px;
-  min-width: 160px;
-  padding: 8px 12px;
+  margin: $nx-spacing-xxs 0 0 0;
+  padding: $nx-spacing-xs 0;
```

Comment:
I think this is supposed to be `$nx-spacing-xxs`

Reply:
> I think this is supposed to be `$nx-spacing-xxs`

Yup: https://github.com/sonatype/sonatype-react-shared-components/pull/55/commits/b9f71421ecf3daa080e0bfcc089414f671876c7e